docs/reference/pip_download.rst

```diff
@@ -42,10 +55,66 @@ Options
 Examples
 ********
 
-#. Download a package and all of its dependencies
-
-    ::
-
-      $ pip download SomePackage
-      $ pip download -d . SomePackage  # equivalent to above
-      $ pip download --no-index --find-links=/tmp/wheelhouse -d /tmp/otherwheelhouse SomePackage
+1. Download a package and all of its dependencies
```

Comment:
Your `::` are breaking the numbered list into severalone-item lists, you should indent the `::` (and their code) more.